### ISSUE TITLE
Auto-Disabling in Gmail mail.google.com

### ISSUE THREAD

**Issue by abhi-soni** [AdguardTeam/AdguardBrowserExtension#2988]
### Please answer the following questions for yourself before submitting an issue

- [X] Filters were updated before reproducing an issue
- [X] I checked the [knowledge base](https://adguard.com/kb/) and found no answer
- [X] I checked to make sure that this issue has not already been filed

### AdGuard Extension version

Version 5.0.128

### Browser version

Chrome Version 129.0.6668.90

### OS version

Windows 11

### Ad Blocking

AdGuard Base filter, AdGuard Quick Fixes filter

### Privacy

AdGuard Tracking Protection filter, AdGuard URL Tracking filter

### Other

Filter unblocking search ads and self-promotion

### What Tracking protection options do you have enabled?

Block trackers, Remove tracking parameters, Ask websites not to track you, Remove X-Client-Data header

### Issue Details

Steps to reproduce:
1. When mail.google.com loads completely, extension showing disabled
2. Enable extension for this website and the page will auto refresh in 3 seconds
3. After refresh finished, the extension shows as disabled.
4. ATTACHED SCREEN RECORDING VIDEO

### Expected Behavior

After enabling extension for mail.google.com then page refreshes. Then as per expected behaviour, this extension should remain enabled until user manually disables it for the website. 

ATTACHED SCREEN RECORDING VIDEO

### Actual Behavior

Initially on mail.google.com the extension shown as disabled. 
So after enabling it for this website the page will auto refresh in 3 seconds 
But after refresh and page load completed, the extension showing as disabled. That's strange as it's auto disabling itself

ATTACHED SCREEN RECORDING VIDEO

### Screenshots

<details></summary>

https://github.com/user-attachments/assets/76f3de74-dbb0-4e2e-90b0-6660e5af0605

</details>

**Comment by AlexandrPkhm** [comment]
@abhi-soni, Hello! Most likely you have [google.com](http://google.com/) domain in the Allowlist. Please remove [google.com](http://google.com/) from the Allowlist and check if the problem persists.

**Comment by abhi-soni** [comment]
@AlexandrPkhm , Yes, as per I checked. google.com was there in the extension allowlist. Not sure when I accidentally added it to there.

Now, upon removing google.com from allowlist, the AdGuard is working perfectly as expected on mail.google.com.
